### ISSUE TITLE
Installation shell script failure: link in installpsh-debian.sh is wrong

### ISSUE THREAD

**Issue by Pawamoy** [PowerShell/PowerShell#5700]
The link to the prod.list file in tools/installpsh-debian.sh is wrong:
https://github.com/PowerShell/PowerShell/blob/master/tools/installpsh-debian.sh#L129

It should not be
```
curl https://packages.microsoft.com/config/ubuntu/$REV/prod.list | $SUDO tee /etc/apt/sources.list.d/microsoft.list
```
but
```
curl https://packages.microsoft.com/config/debian/8/prod.list | $SUDO tee /etc/apt/sources.list.d/microsoft.list
```

Indeed if you go to https://packages.microsoft.com/config/debian, you can see that there is only one directory (`8`), so it's not even needed to use the REV variable, which moreover is set to `"8.9"` on my machine.

**Comment by iSazonov** [comment]
@Pawamoy Thanks for your report!

/cc @TravisEz13 Could you please comment  the Issue?

**Comment by TravisEz13** [comment]
It's usually better to keep the new suggested implementation and the repro separate, but the issue seems accurate.

**Comment by TravisEz13** [comment]
Although, it does not agree with our documentation:  https://github.com/PowerShell/PowerShell/blob/master/docs/installation/linux.md#installation-via-package-repository---debian-8   I would suggest updating to be consistent with the documentation.

**Comment by TravisEz13** [comment]
Debian 9:   https://github.com/PowerShell/PowerShell/blob/master/docs/installation/linux.md#debian-9

**Comment by Pawamoy** [comment]
This has been addressed in #6314.
